js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeExtensionsDisabled.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ai.onnxruntime.reactnative;
+
+import ai.onnxruntime.OrtSession.SessionOptions;
+import android.util.Log;
+
+class OnnxruntimeExtensions {
+  public void registerOrtExtensionsIfEnabled(SessionOptions sessionOptions) {
+    Log.d(
+        "Info",
+        "ORT Extensions is not enabled in the current configuration. If you want to enable this support,"
```

Comment:
nit: add space

```suggestion
        "ORT Extensions is not enabled in the current configuration. If you want to enable this support, "
```

Reply:
also the first argument is a tag, maybe it should be something like "OnnxruntimeExtensions". if you want an info log instead of debug, use `Log.i()`

Reply:
right got it. thanks!